storage/Utils/StorageTmpl.h

```diff
@@ -127,7 +127,7 @@ namespace storage
 	static_assert(!std::is_enum<Type>::value, "is enum");
 
 	if (lhs != rhs)
-	    log << " " << text << ":" << lhs << "-->" << rhs;
+	    log << " " << text << ":" << lhs << "-->" << rhs << "\n";
```

Comment:
change in this file really helps with readability when testsuite failed, as previously lines was mixed together.

Now it returns something like:

```
bcache1.cc(38): error: in "probe": 
sid 52 device differ
 cache_mode:writearound-->writethrough
 writeback_percent:0-->10
 sequential_cutoff:1024-->0
```